spearhead/assets/sass/_header.scss

```diff
@@ -28,6 +28,11 @@ $site-branding-height: 68px;
 	display: flex;
 	justify-content: space-between;
 	max-width: var(--responsive--alignfull-width);
+	padding: var(--global--spacing-horizontal);
```

Comment:
This looks good. I think this is already declared higher in the file but wrapped in `.has-main-navigation`. I think we can remove that one, which will make this simper and dry-er.

Reply:
To do so seems that it will also require changes to Seedlet.  Otherwise the element defers to the more specific selector there which has the 3x margins defined.

Is is less dry but is the smaller change.

Reply:
Oh I see what you mean. In that case 🚢